org.eclipse.jdt.junit.core/src/org/eclipse/jdt/internal/junit/buildpath/BuildPathSupport.java

```diff
@@ -306,51 +306,51 @@ public boolean accept(File dir, String name) {
+			"junit-platform-runner", new VersionRange("[1.0.0,2.0.0)"), null, "junit-platform-runner_1.*.jar", "junit-platform-runner.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
 			JUnitPreferencesConstants.JUNIT_PLATFORM_RUNNER_JAVADOC);
 
 	public static final JUnitPluginDescription JUNIT_PLATFORM_SUITE_API_PLUGIN= new JUnitPluginDescription(
-			"org.junit.platform.suite.api", new VersionRange("[1.0.0,2.0.0)"), null, "org.junit.platform.suite.api_1.*.jar", "org.junit.platform.suite.api.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
+			"junit-platform-suite-api", new VersionRange("[1.0.0,2.0.0)"), null, "junit-platform-suite-api_1.*.jar", "junit-platform-suite-api.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
 			JUnitPreferencesConstants.JUNIT_PLATFORM_SUITE_API_JAVADOC);
 
 	public static final JUnitPluginDescription JUNIT_PLATFORM_SUITE_ENGINE_PLUGIN= new JUnitPluginDescription(
-			"org.junit.platform.suite.engine", new VersionRange("[1.0.0,2.0.0)"), null, "org.junit.platform.suite.engine_1.*.jar", "org.junit.platform.suite.engine.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
+			"junit-platform-suite-engine", new VersionRange("[1.0.0,2.0.0)"), null, "junit-platform-suite-engine_1.*.jar", "junit-platform-suite-engine.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
 			JUnitPreferencesConstants.JUNIT_PLATFORM_SUITE_ENGINE_JAVADOC);
 
 	public static final JUnitPluginDescription JUNIT_PLATFORM_SUITE_COMMONS_PLUGIN= new JUnitPluginDescription(
-			"org.junit.platform.suite.commons", new VersionRange("[1.0.0,2.0.0)"), null, "org.junit.platform.suite.commons_1.*.jar", "org.junit.platform.suite.commons.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
+			"junit-platform-suite-commons", new VersionRange("[1.0.0,2.0.0)"), null, "junit-platform-suite-commons_1.*.jar", "junit-platform-suite-commons.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
 			JUnitPreferencesConstants.JUNIT_PLATFORM_SUITE_COMMONS_JAVADOC);
 
 	public static final JUnitPluginDescription JUNIT_VINTAGE_ENGINE_PLUGIN= new JUnitPluginDescription(
-			"org.junit.vintage.engine", new VersionRange("[4.12.0,6.0.0)"), null, "org.junit.vintage.engine_5.*.jar", "org.junit.vintage.engine.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
+			"junit-vintage-engine", new VersionRange("[4.12.0,6.0.0)"), null, "junit-vintage-engine_5.*.jar", "org.junit.vintage.engine.source", "", //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$//$NON-NLS-5$
 			JUnitPreferencesConstants.JUNIT_VINTAGE_ENGINE_JAVADOC);
 
 	public static final JUnitPluginDescription JUNIT_OPENTEST4J_PLUGIN= new JUnitPluginDescription(
```

Comment:
Can we use this one (JUNIT_OPENTEST4J_PLUGIN) also from maven?

Reply:
Also, JUNIT_APIGUARDIAN_PLUGIN.

Reply:
Or a separate PR is also fine.

Reply:
Yes, this will eventually happen, but at this stage, a separate PR would be better I believe.

Reply:
Sure

Reply:
Actually opentest4j is already only coming from Maven since https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/commit/ca6007d293149936492918827f7a126bd805f411 . apiguardian requires an update in bundle-symbolicName, and this requires first to tweak the .target and then the feature.xml ; so I prefer doing it in a separate PR now (particularly as IIRC some other libs to require apiguardian).